pkg/cmd/install.go

```diff
@@ -230,3 +236,49 @@ func (o *installCmdOptions) waitForPlatformReady(platform *v1alpha1.IntegrationP
 
 	return watch.HandlePlatformStateChanges(o.Context, platform, handler)
 }
+
+func (o *installCmdOptions) validate(cmd *cobra.Command, args []string) error {
+	var result error
+
+	// Let's register only our own APIs
+	schema := runtime.NewScheme()
+	if err := apis.AddToScheme(schema); err != nil {
+		return err
+	}
+
+	if o.contexts == nil {
+		return nil
+	}
+	for _, context := range o.contexts {
+		err := errorIfContextIsNotAvailable(schema, context)
+		result = multierr.Append(result, err)
+	}
+	return result
+}
+
+func errorIfContextIsNotAvailable(schema *runtime.Scheme, context string) error {
+	
+	if context == platform.NoContext {
+		// Indicates that nothing should be installed
```

Comment:
@lburgazzoli should we check also when "none" + another context is given, that this is an error, too ? (as according to the code 'none' takes precedences of anything else ?)

Reply:
yes we should

Reply:
Ok, I will add.

Reply:
added